android/app/src/androidTest/java/com/trafi/example/TestPreviews.kt

```diff
@@ -1,18 +1,10 @@
 package com.trafi.example
 
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.unit.dp
-import com.trafi.ui.Button
 import com.trafi.ui.ButtonPreview
 import com.trafi.ui.ButtonStyledCornerRadiusPreview
-import com.trafi.ui.theme.MaasCornerRadius
-import com.trafi.ui.theme.MaasTheme
 
-val testPreviews = listOf<@Composable () -> Unit> (
-    {
-        ButtonPreview()
-    },
-    {
-        ButtonStyledCornerRadiusPreview()
-    }
+val testPreviews = listOf(
+        arrayOf<Pair<@Composable () -> Unit, String>>(Pair({ButtonPreview()}, "ButtonPreview")),
+        arrayOf<Pair<@Composable () -> Unit, String>>(Pair({ButtonStyledCornerRadiusPreview()}, "ButtonStyledCornerRadiusPreview"))
```

Comment:
Can we write it like this..

```kotlin
val testPreviews: List<Array<Pair<@Composable () -> Unit, String>>> = listOf(
    arrayOf({ ButtonPreview() } to "ButtonPreview"),
    arrayOf({ ButtonStyledCornerRadiusPreview() } to "ButtonStyledCornerRadiusPreview"),
)
```

and what is the purpose of wrapping the `Pair` in an `Array` type?

Reply:
I also tinkered with it a bit, and such case works: 
`package com.trafi.example

import androidx.compose.runtime.Composable
import com.trafi.ui.ButtonPreview
import com.trafi.ui.ButtonStyledCornerRadiusPreview

@Suppress("RemoveExplicitTypeArguments")
val testPreviews = listOf<Pair<@Composable () -> Unit, String>>(
        Pair({ ButtonPreview() }, "ButtonPreview"),
        Pair({ ButtonStyledCornerRadiusPreview() }, "ButtonStyledCornerRadiusPreview")
)`

Reply:
Somehow the `({ ButtonPreview() } to "ButtonPreview")` doesn't work as it throws static code analysis error.